app/src/main/java/br/com/concrete/tentacle/data/models/Game.kt

```diff
@@ -12,7 +12,6 @@ data class Game(
     @SerializedName("id") val remoteGameId: Int? = null,
     @SerializedName("name", alternate = ["title"])
     val name: String = String(),
-    val createdBy: String,
```

Comment:
instead of removing should be Owner

Reply:
Este campo não está sendo usado em nenhum lugar do código. Para evitar erros de parse ele foi removido.